### DISCUSSION TITLE
z plugin does not work in docker container

### DISCUSSION THREAD

**Discussion by qiuwei** [ohmyzsh/ohmyzsh#11340] [Help] [answered]
Got the following error message when typing `z`
```
_setup:37: compstate: assignment to invalid subscript range
```

Have already tried reset the completion system but it did not help.
If I remove the `~/.z`, the plugin works for once but fails after the `~/.z` is automatically generated.

The `omz-debug.log` is attached. The relevant error occurs on line 137741.
[omz-debug.log.tar.gz](https://github.com/ohmyzsh/ohmyzsh/files/10018354/omz-debug.log.tar.gz)

**Comment by agkozak** [comment, marked as answer]
See if my advice [here](https://github.com/ohmyzsh/ohmyzsh/issues/11282) helps.

**Reply by qiuwei** [reply]
It works! Thanks a lot.
